packages/graph/hash_graph/lib/graph/src/store/postgres/query/expression/conditional.rs

```diff
@@ -21,19 +25,60 @@ impl Transpile for Function<'_> {
                 expression.transpile(fmt)?;
                 fmt.write_char(')')
             }
+            Self::JsonExtractPath(paths) => {
+                fmt.write_str("jsonb_extract_path(")?;
+                for (i, expression) in paths.iter().enumerate() {
+                    if i > 0 {
+                        fmt.write_str(", ")?;
+                    }
+                    expression.transpile(fmt)?;
+                }
+                fmt.write_char(')')
+            }
+            Self::JsonContains(json, value) => {
+                fmt.write_str("jsonb_contains(")?;
+                json.transpile(fmt)?;
+                fmt.write_str(", ")?;
+                value.transpile(fmt)?;
+                fmt.write_char(')')
+            }
+            Self::JsonBuildArray(expressions) => {
+                fmt.write_str("jsonb_build_array(")?;
+                for (i, expression) in expressions.iter().enumerate() {
+                    if i > 0 {
+                        fmt.write_str(", ")?;
+                    }
+                    expression.transpile(fmt)?;
+                }
+                fmt.write_char(')')
+            }
+            Self::JsonBuildObject(pairs) => {
+                fmt.write_str("jsonb_build_object(")?;
+                for (i, (key, value)) in pairs.iter().enumerate() {
+                    if i > 0 {
+                        fmt.write_str(", ")?;
+                    }
+                    key.transpile(fmt)?;
+                    fmt.write_str(", ")?;
+                    value.transpile(fmt)?;
+                }
+                fmt.write_char(')')
+            }
         }
     }
 }
 
 #[derive(Debug, PartialEq, Eq, Hash)]
 pub enum Constant {
     Boolean(bool),
+    String(&'static str),
 }
 
 impl Transpile for Constant {
     fn transpile(&self, fmt: &mut fmt::Formatter) -> fmt::Result {
         match self {
             Self::Boolean(value) => fmt.write_str(if *value { "TRUE" } else { "FALSE" }),
+            Self::String(value) => write!(fmt, "'{value}'"),
```

Comment:
Are we relying on `'static` to make sure that user input doesn't get to this place?  I'm just a little cautious of these direct formats :)

Reply:
Yes, that's the reason, why `String` is defined as `Constant::String(&'static str)` 🚀 
Thanks for double-checking this!